website/docs/add-secure-apps/outposts/index.mdx

```diff
@@ -2,6 +2,8 @@
 title: Outposts
 ---
 
+import DocCardList from "@theme/DocCardList";
```

Comment:
On a child-page, let's not use Cards, please. For consistency's sake, we should use them on all top-level landing pages, but not "child" landing pages please. It would be nice to allow the brain to recognize that this is a top-level topic when one sees cards. Give me a good debate if you too feel strongly about using them here, though. :-)